src/Stripe.net/Stripe.net.csproj

```diff
@@ -38,6 +39,12 @@
     <CodeAnalysisRuleSet>..\_stylecop\StyleCopRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.0" />
```

Comment:
This is the backport package that adds support for `IAsyncEnumerable` in .NET Standard 2.0.